src/CacheManager.Serialization.Json/GzJsonCacheSerializer.cs

```diff
@@ -0,0 +1,87 @@
+﻿#if !DOTNET5_2
```

Comment:
Is there any particular reason you exclude .net core? The functionality should be available, even for DOTNET5.2 I think